api/src/data_migration/transformation/subtask/transform_agency.py

```diff
@@ -96,8 +97,14 @@
 class AgencyConfig(PydanticBaseEnvConfig):
     # TODO - we might want to put this somewhere more central
     #        as we might want to filter these out in other places
-    test_agency_config: set[str] = Field(
-        default={"GDIT", "IVV", "IVPDF", "0001", "FGLT", "NGMS", "NGMS-Sub1", "SECSCAN"}
+    prefix_env: str | None = os.getenv("TEST_AGENCY_PREFIXES")
```

Comment:
@chouinar Interested in your thoughts on this approach. The big upside to me is it lets the format in the env var be:
```ABC,DEF,XYZ```
So no risk of breaking the parsing, needing to escape quotes within quotes, etc. It makes the Python code a bit less glamorous but feels much safer/easier to maintain the list going forward.

Reply:
First thing - you don't need to ever use os.getenv, that's what the point of these env configs is, doing that for you (and converting types).
```suggestion
    prefix_env: str | None = Field("GDIT,IVV,IVPDF,0001,FGLT,NGMS,SECSCAN", alias="TEST_AGENCY_PREFIXES")
```

For how to actually parse this into a set, I'd recommend following a pattern like we did for the deploy metadata: https://github.com/HHS/simpler-grants-gov/blob/main/api/src/util/deploy_metadata.py#L38

There's a `model_post_init` function you can setup and Pydantic will run it after it loads everything else. So you could do something like:

```py
    def model_post_init(self, _context: typing.Any) -> None:
        """Run after __init__ sets above values from env vars"""

        self.test_agency_config = set(self.prefix_env.split(","))
```